iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java

```diff
@@ -314,6 +314,8 @@ public interface IManager {
    */
   TSStatus deleteDatabases(List<String> deletedPaths);
 
+  TSStatus createManyDatabases();
```

Comment:
Add java docs to keep the style unified.

Reply:
sure